src/runtime-rs/crates/hypervisor/src/dragonball/inner_device.rs

```diff
@@ -68,6 +72,9 @@ impl DragonballInner {
                 .add_share_fs_device(&sharefs.config)
                 .context("add share fs device"),
             DeviceType::Vsock(_) => todo!(),
+            DeviceType::VhostUserNetwork(dev) => self
+                .add_vhost_user_net_device(&dev.config)
+                .context("add vhost-user-net device"),
```

Comment:
Place the Arm `VhostUserNetwork` before `Vsock`.

Reply:
Fixed.